src/libs/actions/IOU.js

```diff
@@ -689,7 +689,7 @@ function deleteMoneyRequest(chatReportID, iouReportID, moneyRequestAction, shoul
         CONST.IOU.REPORT_ACTION_TYPE.DELETE,
         amount,
         moneyRequestAction.originalMessage.currency,
-        Str.htmlDecode(moneyRequestAction.originalMessage.comment),
+        moneyRequestAction.originalMessage.comment,
```

Comment:
Are we sure this is a plain text?

Reply:
What was the console log for `originalMessage.comment`? Can you check where we set it, and do we set html or text

Reply:
`comment` field simply didn't exist here when I logged. I couldn't find place where comment is set

Reply:
Done